modules/EnsEMBL/Web/Object.pm

```diff
@@ -343,7 +345,7 @@ sub get_slices {
   foreach (@slices) {
     next unless $_;
 
-    my $name = $_->can('display_Slice_name') ? $_->display_Slice_name : $args->{species};
+    my $name = $_->can('display_Slice_name') ? lc($_->display_Slice_name) : $args->{species};
```

Comment:
`lc`?

Reply:
I have no idea where that came from! I remember debugging some stuff to do with compara slices for the covid site, so maybe it crept in then? I'll delete it.